File: backend/Restaurante.Repositorio/Services/Cozinha/Models/ListagemCozinhaModel.cs
using System;
using Restaurante.Dominio.Enum;
using Restaurante.Repositorio.Services.Produto.Models;

namespace Restaurante.Repositorio.Services.Cozinha.Models
{
    public class ListagemCozinhaModel
    {
        public int ComandaId { get; set; }
        public int PedidoId { get; set; }
        public int MesaId { get; set; }
        public ProdutoModel Produto { get; set; }
        public int Quantidade { get; set; }
        public DateTime DataHoraRealizacao { get; set; }
        public DateTime? DataHoraEntrega { get; set; }
        public StatusEnum StatusEnum { get; set; }
    }
}

File: frontend/src/app/pages/cozinha/models/listagem-cozinha.model.ts
import { StatusPedido } from "src/app/consts/status-pedido.enum";
import { ProdutoModel } from "src/app/shared/models/produto.model";

export interface ListagemCozinhaModel {

    comandaId: number;
    pedidoId: number;
    mesaId: number;
    produto: ProdutoModel;
    quantidade:number;
    dataHoraRealizacao: Date;
    dataHoraEntrega: Date;
    statusEnum: StatusPedido;
}